docs/oidc-client-ts.api.md

```diff
@@ -65,8 +65,6 @@ export interface CreateSigninRequestArgs {
     // (undocumented)
     resource?: string;
     // (undocumented)
-    response_mode?: string;
```

Comment:
`response_mode` should not be removed. I do not agree with the statement that it is no longer needed since implicit was removed.

Using response mode `fragment` with code flow is also beneficial as it prevents sending things like "code" to the server which actually serves the application.

Reply:
I would not be able to use `oidc-client-ts` if it should loose support for `fragment` response mode.

Reply:
ok, lets keep this feature